README.md

```diff
@@ -9,124 +9,136 @@ The following repository contains:
+
+```bash
+sudo ./requirementsCheck.sh
+```
 
 ## Installation
 
-We are using macOS to build and run everything locally.
+1. Create a new directory and navigate to it in a terminal.
 
-1. Clone the repositories.
+```bash
+mkdir smartkickers
+cd smartkickers
+```
 
-   ```bash
-   git clone https://github.com/HackYourCareer/SmartKickers.git
-   git clone https://github.com/HackYourCareer/SmartKickersAI.git
-   ```
+2. Clone the repository.
 
-2. Run the [`requirementsCheck.sh`](requirementsCheck.sh) script to check if your system fulfils all requirements.
+```bash
+git clone https://github.com/HackYourCareer/SmartKickersAI.git
+```
 
-   The script will look for open-cv, numpy, imutils, ximer driver only after successfully installing python! So if you don't have python it won't print missing python dependent packages except python itself.
+3. Pull frontend image from GCR
 
-   ```bash
-   cd SmartKickers
-   ./requirementsCheck.sh
-   ```
+```bash
+docker pull ghcr.io/hackyourcareer/smartkickers-frontend:latest
+```
 
-   > **NOTE:** Be sure to give permission for the script to execute.<br/>
+4. Pull backend image from GCR
 
-   ```bash
-   sudo chmod +x requirementsCheck.sh
-   ```
+```bash
+docker pull ghcr.io/hackyourcareer/smartkickers-backend:latest
+```
 
-3. Install missing dependencies.
+## Launching
 
-4. In folder [`SmartKickers`](SmartKickers) run docker-compose build.
+> **NOTE:** Configuration files for the camera view are provided by the developers in the [`SmartKickersAI`](https://github.com/HackYourCareer/SmartKickersAI/tree/main/LocalServer) repository.
 
-   ```bash
-   docker-compose build
-   ```
+### Launching the server and web application
 
-## Launching
+- Run pulled containers
+
+  - In a terminal run:
 
-> **NOTE:** Configuration files for the camera view are provided by the developers in [`SmartKickersAI`](https://github.com/HackYourCareer/SmartKickersAI/tree/main/LocalServer) repository.
+  ```bash
+  docker run -p 3000:3000 ghcr.io/hackyourcareer/smartkickers-backend:latest
+  ```
+
+  - Open a new terminal and run:
 
-1. In the folder [`SmartKickers`](SmartKickers) launch the docker-compose.
+  ```bash
+  docker run -p 3007:80 ghcr.io/hackyourcareer/smartkickers-frontend:latest
+  ```
 
-   ```bash
-   docker-compose up
-   ```
+### Launching the ball-tracking software
 
-   After launching you will see that the backend and the frontend of the application has started successfully.
+1. Plug in the camera to the computer.
 
-   <h1 align="center">
-       <img src="assets/appLaunch.png">
-   </h1>
+2. In a new terminal window, launch the Node.js server.
```

Comment:
I know you've changed this point in the meantime to: 
```
2. In order to launch the application, navigate to the `SmartKickersAI` repo and double click `start` file.
   Successful connection to the Go server shows the connection details like in the screenshot below:
   ![Node launch image](assets/nodeLaunch.png "Node launch")
```

So I'll refer to the updated one instead. 

- `In order to run` and `To run` express the same thing, so it's better to go with the simpler one (remember that your users are not native English speakers, so it's best that the language is simple and doesn't stand in the way of using your product.) 
- An article is missing before the file name. 
  - Also, **shouldn't there be an extension in this file name?** I can't check it 'cause I don't have access to that repo. 
- You shouldn't add screenshots with code snippets/terminal output to documentation. Your user might have the shell skin formatted differently to yours and the formatting might actually add cognitive load. Instead, copy the output and add it as a code block.
  - Also, is it still relevant? In the screenshot, you can see the command that you previously had in your documentation to launch the application, but now you click the `start` file. 

``````suggestion
2. To launch the application, navigate to the `SmartKickersAI` repo and double click the `start` file.
   On successful connection to the Go server, you see the following output:
  ```bash
    {
      url: "ws://127.0.0.1:3000",
      wsurl: "ws://127.0.0.1:3000/shot",
      dispatcherlURL: 'ws://127.0.0.1:3000'
      cfURL: 'ws://127.0.0.1:3000/cf'
      tableID: '10'
      token: 'c3d8c29ec43b9eadb8bc80ad1458ab8'
      messageTypes: {
          BallPosition: '615eec77d6be09356891'
          Shot: '2ca34aee4bb3dec060b1'
          BestShotGuid: 'c01b38461a678f3eefa5'
      } 
  }
  init
  conn!
  { type: 'utf8', utf8Data: '{"start":"10"}' }
  { start: '10' }
  game id
  10
  connected to IoT Services via wss
  ```
``````

Also, if you're the ones passing the URLs, I'd make the strings consistent and use either `'`s, or `"`s, not both.